promgen/views.py

```diff
@@ -350,6 +350,16 @@ def post(self, request, pk):
         return JsonResponse({'redirect': exporter.project.get_absolute_url()})
 
 
+class NotifierToggle(LoginRequiredMixin, View):
+    def post(self, request, pk):
+        sender = get_object_or_404(models.Sender, id=pk)
+        sender.enabled = not sender.enabled
+        sender.save()
+        signals.trigger_write_config.send(request)
```

Comment:
No need to trigger writing Prometheus' configuration since Notifiers are completely internal to Promgen

Reply:
fixed